app/views/schools/your_school_estates/edit.html.erb

```diff
@@ -21,28 +21,64 @@
-        <%= f.input :alternative_heating_lpg_notes, label: t('schools.your_school_estates.edit.alternative_heating_sources.notes') %>
+        <%= f.input :alternative_heating_lpg_percent,
+                    label: t('.alternative_heating_sources.lpg.percentage') %>
+        <%= f.input :alternative_heating_lpg_notes,
+                    label: t('.alternative_heating_sources.notes') %>
       </div>
 
-      <%= f.input :alternative_heating_biomass, label: t('schools.your_school_estates.edit.alternative_heating_sources.biomass.label'), input_html: {data: {reveals: '.school_alternative_biomass_fields' }} %>
+      <%= f.input :alternative_heating_biomass,
+                  label: t('.alternative_heating_sources.biomass.label'),
+                  input_html: { data: { reveals: '.school_alternative_biomass_fields' } } %>
       <div class='school_alternative_biomass_fields' data-revealed-by=".school_alternative_heating_biomass">
-        <%= f.input :alternative_heating_biomass_percent, label: t('schools.your_school_estates.edit.alternative_heating_sources.biomass.percentage') %>
-        <%= f.input :alternative_heating_biomass_notes, label: t('schools.your_school_estates.edit.alternative_heating_sources.notes') %>
+        <%= f.input :alternative_heating_biomass_percent,
+                    label: t('.alternative_heating_sources.biomass.percentage') %>
+        <%= f.input :alternative_heating_biomass_notes,
+                    label: t('.alternative_heating_sources.notes') %>
       </div>
 
-      <%= f.input :alternative_heating_district_heating, label: t('schools.your_school_estates.edit.alternative_heating_sources.district_heating.label'), input_html: {data: {reveals: '.school_alternative_district_heating_fields' }} %>
+      <%= f.input :alternative_heating_district_heating,
+                  label: t('.alternative_heating_sources.district_heating.label'),
+                  input_html: { data: { reveals: '.school_alternative_district_heating_fields' } } %>
       <div class='school_alternative_district_heating_fields' data-revealed-by=".school_alternative_heating_district_heating">
-        <%= f.input :alternative_heating_district_heating_percent, label: t('schools.your_school_estates.edit.alternative_heating_sources.district_heating.percentage') %>
-        <%= f.input :alternative_heating_district_heating_notes, label: t('schools.your_school_estates.edit.alternative_heating_sources.notes') %>
+        <%= f.input :alternative_heating_district_heating_percent,
+                    label: t('.alternative_heating_sources.district_heating.percentage') %>
+        <%= f.input :alternative_heating_district_heating_notes,
+                    label: t('.alternative_heating_sources.notes') %>
+      </div>
+
+      <%= f.input :alternative_heating_ground_source_heat_pump,
+                  label: t('.alternative_heating_sources.ground_source_heat_pump.label'),
```

Comment:
Had to change all the i18n keys in this section to be relative rather than absolute in order to please erblint line length